config/hooks/post-merge

```diff
@@ -10,22 +10,26 @@ trap "rm -f ${files_changed_upstream}" EXIT
 
 git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD > "$files_changed_upstream"
 
-grep -E --quiet 'requirements.txt' "$files_changed_upstream" && needs_py_update=1
-
-grep -E --quiet 'migrations/groups.py' "$files_changed_upstream" && needs_update=1
-
-[[ "$needs_update" ]] && cat <<EOF
+grep -E --quiet 'migrations/groups.py' "$files_changed_upstream" && cat <<EOF
 
 [${red}${bold}!!!${reset}] Some migrations have changed, you may need to run:
 
   snuba migrations migrate --force
 
 EOF
 
-[[ "$needs_py_update" ]] && cat <<EOF
+grep -E --quiet 'requirements.txt' "$files_changed_upstream" && cat <<EOF
 
 [${red}${bold}!!!${reset}] Python dependencies updated, you may need to run:
 
   make install-python-dependencies
 
 EOF
+
+grep -E --quiet 'rust_snuba/' "$files_changed_upstream" && cat <<EOF
+
+[${red}${bold}!!!{$reset}] Rust code changed, you may need to run:
+
+  make install-rs-dev
+
+EOF
```

Comment:
Can we avoid ever requiring the user to do this? What if the `snuba rust-consumer` CLI command just runs it before launching the consumer instead?

Reply:
@lynnagara i see you already approved the PR, but i think we could do this. the concern i'd see is that when we change the rust code, people will be forced to recompile everytime, even though they might be fine running an older version of the code

for example, i also sometimes skip on the `make install-python-dependencies` since the code likely runs fine for my purposes anyway